src/network/networkd-wwan-bus.c

```diff
@@ -0,0 +1,362 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "bus-error.h"
+#include "bus-map-properties.h"
+#include "bus-parse-xml.h"
+#include "bus-util.h"
+#include "networkd-manager.h"
+#include "networkd-wwan-bus.h"
+#include "networkd-wwan.h"
+
+/* From ModemManager-enums.h */
+typedef enum {
+    MM_BEARER_IP_FAMILY_NONE    = 0,
+    MM_BEARER_IP_FAMILY_IPV4    = 1 << 0,
+    MM_BEARER_IP_FAMILY_IPV6    = 1 << 1,
+    MM_BEARER_IP_FAMILY_IPV4V6  = 1 << 2,
+    MM_BEARER_IP_FAMILY_ANY     = 0xFFFFFFFF
+} MMBearerIpFamily;
```

Comment:
Wouldn't it be better to just reuse `ADDRESS_FAMILY_*` for this? The mapping seems to be 1:1 anyway.